### PR TITLE
Fixes #21 and Fixes the logout bug

### LINKED ISSUE #21: name in edit profile is not prepopulated correctly
image

### PR DESCRIPTION
**Pull request by rhnprabhune**

### CODE REVIEW

**Comment by rhnprabhune** [comment]
@krishnasaurabh Have implemented the logout functionality which fixes #9 ,as well fixed #21. Can you please check if its implemented Correctly ?

File: Controller/home.py [hunk 1]
```diff
@@ -4,7 +4,6 @@
 from werkzeug.utils import redirect
 from Controller.user_controller import User
 from Controller.application_controller import Application
-from Controller.email_framework import *
```
Comment (krishnasaurabh) [inline Controller/home.py:7]: Please only import necessary functions and classes, as it would pollute the name space

File: Controller/home.py [hunk 2]
```diff
@@ -41,10 +40,12 @@ def login():
 
 @home_route.route('/auth', methods=['GET'])
 def auth():
-    data = user.get_auth_user_dao(session['email'])
-    data["wishlist"] = application.get(session['email'], '')
-    return render_template('home.html', data=data, upcoming_events=upcoming_events)
-
+    if 'email' in session: 
+        data = user.get_auth_user_dao(session['email'])
+        data["wishlist"] = application.get(session['email'], '')
```
Comment (krishnasaurabh) [inline Controller/home.py:45]: I dont think it just populates the wishlist, it gets all the applications of the user, can you please create a bug to change the name here.

File: Controller/home.py [hunk 3]
```diff
@@ -116,7 +117,6 @@ def add_new_application():
         error = "This job application could not be stored in the database. Please try again."
         return render_template('home.html', jobAddError=error)
     data = {}
-    s_email(company_name,location,job_profile,salary,username,password,session['email'],security_question,security_answer,notes,date_applied,status)
```
Comment (krishnasaurabh) [inline Controller/home.py:119]: Code formatting needs to be done, Please create bug for implementing black, auto black, assign @arjunptm

File: Controller/home.py [hunk 4]
```diff
@@ -131,7 +131,6 @@ def change_status_application():
         error = "This job application could not be stored in the database. Please try again."
         return render_template('home.html', jobAddError=error)
     data = {}
-    status_change_email(application_id,session['email'],status)
```
Comment (krishnasaurabh) [inline Controller/home.py:134]: Is this deletion by mistake or intentional, as it doesn't seem to be related.

**Comment by krishnasaurabh** [comment]
Please review the comments and make the necessary changes.

File: Controller/home.py (hunk 1 above)
Comment (rhnprabhune) [reply Controller/home.py:7]: This was redundant. Hence was removed.

File: Controller/home.py (hunk 2 above)
Comment (rhnprabhune) [reply Controller/home.py:45]: Yes. Will look into it

File: Controller/home.py (hunk 4 above)
Comment (rhnprabhune) [reply Controller/home.py:134]: @krishnasaurabh  These seem to be done by mistake. I will roll back the changes and test the functionalities again

**Comment by krishnasaurabh** [comment]
What is the status on this?

**Comment by rhnprabhune** [comment]
@krishnasaurabh  Looks like I didn't modify the latest code. I have fixed the issue. Functionalities working fine. Good to merge ?

**Review by krishnasaurabh — APPROVED** [review]
